src/api/logs.ts

```diff
@@ -30,6 +30,51 @@ export async function listLogResources(
   }
 }
 
+/**
+ * Calls the API to retrieve a list of all assets
+ *
+ * @param {Sid} environmentSid environment in which to get logs
+ * @param {Sid} serviceSid service to look for logs
+ * @param {GotClient} client API client
+ * @returns {Promise<LogApiResource[]>}
+ */
+export async function listOnePageLogResources(
+  environmentSid: Sid,
+  serviceSid: Sid,
+  client: GotClient,
+  pageSize: number = 50,
+  functionSid?: string,
```

Comment:
Minor nitpick but I'm wondering if this part should be an `opts` object instead. Say you want to filter by startDate but not by functionSid